vip-helpers/vip-utils.php

```diff
@@ -1077,6 +1077,41 @@ function wpcom_vip_plugin_is_loaded( $plugin ) {
 	return in_array( $plugin, wpcom_vip_get_loaded_plugins() );
 }
 
+/**
+ * Load `vipgo-helper.php` if it exists for a network-activated plugin
+ *
+ * Technically tries to include the main plugin file again, but we don't care, because it uses `include_once()` and is called after Core loads the plugin
+ */
+function wpcom_vip_load_helpers_for_network_active_plugins() {
+	// wp_get_active_network_plugins() won't exist otherwise
+	if ( ! is_multisite() ) {
+		return;
+	}
+
+	foreach ( wp_get_active_network_plugins() as $plugin ) {
+		_wpcom_vip_include_plugin( $plugin );
+	}
+}
+add_action( 'muplugins_loaded', 'wpcom_vip_load_helpers_for_network_active_plugins' );
```

Comment:
Correct, since we aren't doing anything else related to plugin-loading on this action.